src/current.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable  max-len, no-restricted-syntax, guard-for-in, quotes, no-dupe-keys, camelcase, no-alert, no-trailing-spaces, no-plusplus,  import/no-cycle */
+import { Key } from './index';
+import { Logic } from './logic';
+
+class CurrentLocation {
+  static getCo_ordinate() {
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(this.myLocation);
+    } else {
+      alert('Geolocation is not supported by this browser! Press okey to continue');
+    }
+  }
+
+  static myLocation(position) {
+    const lat = position.coords.latitude;
+    const long = position.coords.longitude;
+
+    async function getLocation() {
+      const response = await fetch(
+        `https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${long}&appid=${Key.theKey()}`,
+        {
+          mode: 'cors',
+        },
+
+      );
```

Comment:
- **_It should have error handling for `promises` or `async/await` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))_**